onnxmltools/utils/float16_converter.py

```diff
@@ -0,0 +1,113 @@
+import numpy as np
+from struct import unpack
+from ..convert.common.data_types import *
+
+
+def _npfloat16_to_int(np_list):
+    """
+    Convert numpy float16 to python int.
+
+    :param np_list: numpy float16 list
+    :return int_list: python int list
+    """
+    return [int(bin(_.view('H'))[2:].zfill(16), 2) for _ in np_list]
+
+
+def float16_converter(model):
+    """
+    Convert ONNX model from TensorProto type float to float16.
```

Comment:
Maybe you can use this help message: Convert floats in the specified ONNX model to float16.